Bloxstrap/UI/Elements/Menu/Pages/FastFlagEditorPage.xaml.cs

```diff
@@ -253,24 +259,41 @@ private bool ValidateFlagEntry(string name, string value)
             string lowerValue = value.ToLowerInvariant();
             string errorMessage = "";
 
-            if (!_validPrefixes.Where(x => name.StartsWith(x)).Any())
+            if (!_validPrefixes.Any(name.StartsWith))
                 errorMessage = Bloxstrap.Resources.Strings.Menu_FastFlagEditor_InvalidPrefix;
             else if (!name.All(x => char.IsLetterOrDigit(x) || x == '_'))
                 errorMessage = Bloxstrap.Resources.Strings.Menu_FastFlagEditor_InvalidCharacter;
+            
+            // sorry
+            if (name.EndsWith("_PlaceFilter") || name.EndsWith("_DataCenterFilter"))
+                errorMessage = !Validate_Filter(name, value) ? Bloxstrap.Resources.Strings.Menu_FastFlagEditor_InvalidPlaceFilter : ""; 
             else if ((name.StartsWith("FInt") || name.StartsWith("DFInt")) && !Int32.TryParse(value, out _))
                 errorMessage = Bloxstrap.Resources.Strings.Menu_FastFlagEditor_InvalidNumberValue;
             else if ((name.StartsWith("FFlag") || name.StartsWith("DFFlag")) && lowerValue != "true" && lowerValue != "false")
                 errorMessage = Bloxstrap.Resources.Strings.Menu_FastFlagEditor_InvalidBoolValue;
 
+            
             if (!String.IsNullOrEmpty(errorMessage))
-            {
+            { 
                 Frontend.ShowMessageBox(String.Format(errorMessage, name), MessageBoxImage.Error);
                 return false;
             }
 
             return true;
         }
 
+        private bool Validate_Filter(string name, string value)
```

Comment:
Can you remove the underscore from the function name? We do pascal case for function names.